index.js

```diff
@@ -5,34 +5,29 @@ var root = doc && doc.documentElement;
 var cookie = require('cookie-cutter');
 var ua = require('ua-parser-js');
 var userLang = navigator.language.slice(-2) || navigator.userLanguage.slice(-2) || 'us';
+var iosUrl = outlookCom.iOSUrl;
+var andUrl = outlookCom.androidUrl;
+var bannerShow = true;
 
 // platform dependent functionality
 var mixins = {
 	ios: {
-		appMeta: 'apple-itunes-app',
 		iconRels: ['apple-touch-icon-precomposed', 'apple-touch-icon'],
 		getStoreLink: function() {
-			return 'https://itunes.apple.com/' + this.options.appStoreLanguage + '/app/id' + this.appId;
+			return iosUrl;
 		}
 	},
 	android: {
-		appMeta: 'google-play-app',
 		iconRels: ['android-touch-icon', 'apple-touch-icon-precomposed', 'apple-touch-icon'],
 		getStoreLink: function() {
-			return 'http://play.google.com/store/apps/details?id=' + this.appId;
-		}
-	},
-	windows: {
-		appMeta: 'msApplication-ID',
-		iconRels: ['windows-touch-icon', 'apple-touch-icon-precomposed', 'apple-touch-icon'],
-		getStoreLink: function() {
-			return 'http://www.windowsphone.com/s?appid=' + this.appId;
+			return andUrl;
 		}
 	}
 };
 
 var SmartBanner = function(options) {
 	var agent = ua(navigator.userAgent);
+	bannerShow = outlookCom.showBanner;
```

Comment:
@jbbuckner actually, sorry I forgot what I had mentioned earlier. We can toggle whether to show the banner or not by just not call new SmartBanner in the Outlook JavaScript.